### DISCUSSION TITLE
Highways? already implemented?

### DISCUSSION THREAD

**Discussion by githubfan1** [Keriew/augustus#714] [General]
I found the new highway feature. People move really fast but I can't plot any building touching the highway. It seems the highway is still under development. Let's see how it will became.

**Comment by PrettyFlower** [comment]
This is intentional. Highways should be pretty much functional at this point, we're mostly just looking for feedback. I don't think I've written it out yet, but the rules for highways are:
- Walkers on highways move twice as fast.
- Only destination walkers can use highways (caravans, immigrants, soldiers, cart pushers, market ladies going to granaries, etc). Roamers can't use them.
- Walkers will favor the right side of the highway (relative to the walker, not the screen), but will have to cross over to make left turns and such.
- Buildings do not get road access from highways as we didn't want highways to completely replace roads.
- Highways can only be built/cleared in 2x2 tile increments to ensure they take up more space than roads. Highways can partially overlap other highways so that they can be made to line up properly.
- Gatehouses and aqueducts can be built over highways.
- Highways cost 100 dn per 2x2 tiles and have a levy tax of 1 dn per 2x2 tiles per month.
- Highways are not desirable. People do not want to live near them.
- Roads will be automatically paved within 3 tiles of a highway for better graphical transitions.

Here's a test city someone made with highways:
![zardo_valentia_highways2](https://user-images.githubusercontent.com/2081059/195959213-3b794acf-6b7e-4651-b8b6-a5974274b87c.png)

**Reply by githubfan1** [reply]
Thank you for explaining this new feature. 
Aqueducts built on highways do not stop people walking under.
$100 bucks for a 2x2 seems very expensive (more than the cost of a forum).
I liked it very much specially to move quickly troops from one extreme of the map to the other end.

![highway-aqueduct](https://user-images.githubusercontent.com/6599012/195964679-52b60860-6e2b-4e31-996f-8833a2b67750.png)

**Reply by PrettyFlower** [reply]
Thanks for the feedback!

Aqueducts should allow people to walk under them (just like on roads), but obviously you shouldn't be able to string them along highways as you've done in your screenshot. Someone else noticed that as well and I'm working on a fix.

Currently they're pretty expensive because there's some debate as to whether or not they're "too good". All of the existing missions and maps were designed to be beaten without a way for workers to move twice as fast, so making them very expensive prevents players from getting them too early in the game.

**Reply by andythegame** [reply]
Hello Guys,

I've downloaded the latest version, it seem that there are all the new features but I can't find highways. Someone could help me please?

**Reply by crudelios** [reply]
If you're using the latest unstable, it will be under the "Road" button.

**Reply by andythegame** [reply]
Thank you Crudelios, not I'm using the normal one, I have to download the unstable to find the highways?

**Reply by crudelios** [reply]
Exactly, it's only available on the unstable build.

Beware though: it's actually pretty unstable lately due to the amount of new features that have recently been added!

**Comment by githubfan1** [comment]
Now that the road button is not longer available in one click I found very unconfortable to open the submenu to find the road icon. Somehow we need a new button just for the road standalone since it is used very often.

**Reply by PrettyFlower** [reply]
Yeah…this is another one that’s become a source of debate (you should really join the discord!). So far we’ve had one person agree with you and say that because roads are used more frequently they should get their own button and one person say that road blocks and bridges should actually be moved into the road submenu as well. Personally I’m for keeping roads as a submenu since I think there’s too many items in some of the other ones (government, industry, engineering) and the extra click really doesn’t bother me.

**Reply by MHCreations** [reply]
pressing the "R" button brings up the road placement tile. I always find this quicker than selecting road from the menu.

**Reply by githubfan1** [reply]
> pressing the "R" button brings up the road placement tile. I always find this quicker than selecting road from the menu.

"R" is used to rotate an image when plotting a building. Comma "," is used to draw a road.
